src/main/resources/data-repository-openapi.yaml

```diff
@@ -2658,6 +2692,56 @@ definitions:
         items:
           $ref: '#/definitions/PolicyModel'
 
+  DataDeletionRequest:
+    description: |
+      a container holding a specification for how to delete tabular data in one or more tables
+    type: object
+    required:
+    - specType
+    - deleteType
+    properties:
+      deleteType:
+        type: string
+        enum:
+        - soft
+      specType:
+        type: string
+        enum:
+        - gcsFile
+      tables:
+        type: array
+        items:
+          $ref: '#/definitions/DataDeletionTableModel'
+
+  DataDeletionTableModel:
+    description: |
+      a specification for how to delete tabular data in one table
+    type: object
+    required:
+      - tableName
+    properties:
+      tableName:
+        type: string
+        description: the name of a table in the dataset
+      gcsFileSpec:
+        $ref: '#/definitions/DataDeletionGcsFileModel'
+
+  DataDeletionGcsFileModel:
```

Comment:
This is not yet implemented, right?  Usually we don't put APIs in here on spec.

Reply:
DanM had mentioned these names are confusing, and I think this is confirmation of it :)

This model is used, it is the "object that contains a path property with a gs URI that contains row ids for a specific table to be soft deleted". I'm having trouble finding a concise name.

Maybe switching to `TabularDataDeletion` for all of these prefixes, and instead of calling this a `GcsFile` it can be a `GcsUri` to avoid confusion with file deletions? That would leave this: `TabularDataDeletionGcsUriModel`